### ISSUE TITLE
Payment fails on payment in second prestashop (multistore)

### ISSUE THREAD

**Issue by blupman** [mollie/PrestaShop#199]
### Expected behavior
I have multiple webshops running in 1 prestashop. we use multistore.
(http://doc.prestashop.com/display/PS17/The+multistore+interface)

we expect payments to work from all stores.

### Actual behavior
the first shop we configured mollie does work, payments work fine.

in the second shop, selecting payment method ideal, results in a 500 error:
on this url. /module/mollie/payment?method=ideal&rand=1597363119

we uninstalled the mollie module, and configured the other shop first, and than that one work, and the second one stopped working.

actually this did work in the 3.something modules. but were upgrading to prestashop 1.7

<img width="1393" alt="Screenshot 2020-08-14 at 02 14 30" src="https://user-images.githubusercontent.com/4933346/90199073-ed761c00-ddd3-11ea-8fc5-b7ecf3a2fbce.png">

### Steps to reproduce the behavior
1. create 2 or more  shops in the same same shop group. (we actualy have 4)
2. install mollie module.
3. install a different api key in each store.
4. do a test payment.

### Environment
* PrestaShop version: 1.7.6.7
* Mollie module version: 4.0.7
* PHP version: 7.1.33

**Comment by SimonasB88** [comment]
Hey @blupman thanks for your message!

We have backlogged this case and we will let you know about the updates!

Best Regards.

**Comment by SimonasB88** [comment]
Hey @blupman , we tried to replicate the case, but unfortunately we couldn't do this on clean Prestashop 1767, PHP 7.1 2 Multistores on one shop group and using Mollie v.4.0.7.

Tried with different live and test API's on different multishops, 
- shop1 test API
- shop2 test API
- shop3 live API 
but the checkouting seems to be finishing OK with all.

Make sure, that all the checkboxes down the module configuration UI are marked.
![image](https://user-images.githubusercontent.com/29095235/90397718-f2232480-e0a0-11ea-88c6-a6681e7a9617.png)

However it could be an issue, regarding the installation process. 

You should configure the multistores of Prestashop first, then choose to install Mollie module on All Shops mode, not to install separately on each shop. Prestashop should divide the dependencies of the module to all shops.

Please try this suggestion, if else happens, we will take other actions as well.

Best Regards.

**Comment by blupman** [comment]
Thanks for the feedback.

We downgraded the module to the latest 3 version of the mollie module, and we got it to work.

It needs to be mentioned that the prestashop does contain a lot of older products,customers and orders, since were in the process of upgrading from 1.6 to 1.7. 
We didn't perform the auto-upgrade, but we used a migration module to only transfer the needed data.

I will remove to 3 module and try again according to your instructions, and update this issue afterwards. 

Thanks for the help!
